midend/flattenInterfaceStructs.cpp

```diff
@@ -113,28 +169,40 @@ const IR::Node* ReplaceStructs::postorder(IR::Type_Struct* type) {
 }
 
 const IR::Node* ReplaceStructs::postorder(IR::Member* expression) {
+    LOG3("EXPR: " << expression);
     // Find out if this applies to one of the parameters that are being replaced.
-    if (getParent<IR::Member>() != nullptr)
+    if (getParent<IR::Member>() != nullptr) {
+      //        std::cout << "Parent not null: " << std::endl;
```

Comment:
please delete commented-out code.

Reply:
All your changes to this function are just additional logging code.

Reply:
> please delete commented-out code.

Has been deleted.

Reply:
> All your changes to this function are just additional logging code.

Right.  Have cleaned up code because now we have a separate ReplaceHeader class in code.